src/pages/PullRequestPage/PullCoverage/routes/FileExplorer/FileExplorer.spec.tsx

```diff
@@ -19,115 +18,147 @@ const queryClient = new QueryClient({
     pull: {
       head: {
         commitid: '123',
         pathContents: {
-          results: [],
           __typename: 'UnknownPath',
+          message: 'unknown path',
         },
       },
     },
+    repositoryConfig: {
+      indicationRange: {
+        upperRange: 80,
+        lowerRange: 60,
+      },
+    },
   },
 }
 
 const mockMissingCoverage = {
-  username: 'nicholas-codecov',
   repository: {
+    __typename: 'Repository',
     pull: {
       head: {
         commitid: '123',
         pathContents: {
-          results: [],
           __typename: 'MissingCoverage',
+          message: 'missing coverage',
         },
       },
     },
+    repositoryConfig: {
+      indicationRange: {
+        upperRange: 80,
+        lowerRange: 60,
+      },
+    },
   },
 }
 
 const mockListData = {
-  username: 'nicholas-codecov',
   repository: {
+    __typename: 'Repository',
     pull: {
       head: {
         commitid: '123',
         pathContents: {
+          __typename: 'PathContents',
           results: [
             {
               __typename: 'PathContentFile',
+              name: 'file.js',
+              path: 'a/b/c/file.js',
               hits: 9,
-              misses: 0,
+              misses: 1,
               partials: 0,
               lines: 10,
-              name: 'file.js',
-              path: 'a/b/c/file.js',
-              percentCovered: 100.0,
+              percentCovered: 90.0,
+              isCriticalFile: false,
             },
           ],
-          __typename: 'PathContents',
         },
       },
     },
+    repositoryConfig: {
+      indicationRange: {
+        upperRange: 80,
+        lowerRange: 60,
+      },
+    },
   },
 }
 
 const mockTreeData = {
-  username: 'nicholas-codecov',
   repository: {
+    __typename: 'Repository',
     pull: {
       head: {
         commitid: '123',
         pathContents: {
           results: [
             {
               __typename: 'PathContentDir',
+              name: 'src',
+              path: 'src',
               hits: 9,
-              misses: 0,
+              misses: 1,
               partials: 0,
               lines: 10,
-              name: 'src',
-              path: 'src',
-              percentCovered: 100.0,
+              percentCovered: 90.0,
             },
             {
               __typename: 'PathContentFile',
+              name: 'file.js',
+              path: 'a/b/c/file.js',
               hits: 9,
-              misses: 0,
+              misses: 1,
               partials: 0,
               lines: 10,
-              name: 'file.js',
-              path: 'a/b/c/file.js',
-              percentCovered: 100.0,
+              percentCovered: 90.0,
+              isCriticalFile: false,
             },
           ],
           __typename: 'PathContents',
         },
       },
     },
+    repositoryConfig: {
+      indicationRange: {
+        upperRange: 80,
+        lowerRange: 60,
+      },
+    },
   },
 }
 
 const wrapper =
```

Comment:
nit: I think we prefer the patter of React.FC<PropsWithChildren> instead of ReactNode